boltdesignsystem/bolt
# Hotfix: paragraph spacing

## Summary

Fixes a bug where a `<p>` as last child is generating undesired white space.

## Details

1. Re-organized the generic folder
2. Put in the proper reset for last-child elements
3. Removed certain redundant rules

## How to test

Run the branch locally and create a few plain `<p>` elements on a page, make sure the last-child does not have any margin-bottom.
